src/ChangeLog:

2006-05-02  John W. Eaton  <jwe@octave.org>

	* lex.l ({CCHAR}): Exit rawcommand mode if returning '\n' token.


Index: src/lex.l
===================================================================
RCS file: /cvs/octave/src/lex.l,v
retrieving revision 1.232
diff -u -r1.232 lex.l
--- src/lex.l	24 Apr 2006 19:13:08 -0000	1.232
+++ src/lex.l	2 May 2006 23:31:14 -0000
@@ -696,7 +696,10 @@
       BEGIN (INITIAL);
 
     if (nesting_level.none ())
-      COUNT_TOK_AND_RETURN ('\n');
+      {
+	lexer_flags.doing_rawcommand = false;
+	COUNT_TOK_AND_RETURN ('\n');
+      }
     else if (nesting_level.is_bracket_or_brace ())
       COUNT_TOK_AND_RETURN (';');
   }
